### PR TITLE
chore: publish temporary version of inquirer

### PR DESCRIPTION
**Pull request by peternhale**
@W-9670281@

### CODE REVIEW

**Comment by uip-robot-zz** [comment]
Git2Gus App is installed but the `.git2gus/config.json` doesn't exist.

**Review by RodEsp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,10 +2,11 @@
 
 # Inquirer.js
 
-[![npm](https://badge.fury.io/js/inquirer.svg)](http://badge.fury.io/js/inquirer)
-[![tests](https://travis-ci.com/SBoudrias/Inquirer.js.svg?branch=master)](https://app.travis-ci.com/github/SBoudrias/Inquirer.js)
-[![Coverage Status](https://codecov.io/gh/SBoudrias/Inquirer.js/branch/master/graph/badge.svg)](https://codecov.io/gh/SBoudrias/Inquirer.js)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FSBoudrias%2FInquirer.js.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2FSBoudrias%2FInquirer.js?ref=badge_shield)
+Salesforce fork of [SBoudrias/Inquirer.js](https://github.com/SBoudrias/Inquirer.js)
```
Comment (jshackell-sfdc) [inline README.md]:
```suggestion
Salesforce fork of [SBoudrias/Inquirer.js](https://github.com/SBoudrias/Inquirer.js).
```

File: README.md [hunk 2]
```diff
@@ -2,10 +2,11 @@
...
+
+Please post any issues related to this fork to [salesforcecli/cli](https://github.com/salesforcecli/cli/issues)
```
Comment (jshackell-sfdc) [inline README.md]:
```suggestion
Post any issues related to this fork to [salesforcecli/cli](https://github.com/salesforcecli/cli/issues).
```

File: README.md [hunk 3]
```diff
@@ -2,10 +2,11 @@
...
+
+This module is temporary and may be removed at anytime. Please **DO NOT** use this module as direct dependency.
```
Comment (jshackell-sfdc) [inline README.md]:
```suggestion
This module is temporary and may be removed at any time. **DO NOT** use this module as a direct dependency.
```

**Pull request merged**